patches/jsdom@20.0.3.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/lib/jsdom/living/post-message.js b/lib/jsdom/living/post-message.js
+index db16576db84d3aa45fe8469b4325447c945097d7..1fe3890dcf7cf22f48a62bd8e2bafa41f85cb1e1 100644
+--- a/lib/jsdom/living/post-message.js
++++ b/lib/jsdom/living/post-message.js
+@@ -6,7 +6,7 @@ const { isValidTargetOrigin } = require("../utils");
+ const { fireAnEvent } = require("./helpers/events");
+ 
+ module.exports = function (globalObject) {
+-  return function (message, targetOrigin) {
++  return function (message, targetOrigin, ports) {
```

Comment:
We use the third parameter of `postMessage` in our code to create the private channel between the host application and the Custom View but `jsdom` does not support it yet:
* [Here](https://github.com/jsdom/jsdom/blob/master/lib/jsdom/living/post-message.js#L33) is where they have a TODO about it
* There's [this issue](https://github.com/jsdom/jsdom/issues/2448) asking for its support
* [Here's a PR](https://github.com/jsdom/jsdom/pull/3521) trying to implement this missing functionality (already 4 months old)

Since our case is very basic, I implemented this simplistic patch so it works in our tests.

Reply:
Not needed anymore as we are now mocking this part in a different way (mock the providers instead of relying on the communication to happen in the tests environment).

Just leave the comment here for reference purposes.